inc/compatibilities/sassy_social_share.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * Class Optml_sassy_social_share.
+ *
+ * @reason Sassy social share picks eco images to share
+ */
+class Optml_sassy_social_share extends Optml_compatibility {
+	/**
+	 * Should we load the integration logic.
+	 *
+	 * @return bool Should we load.
+	 */
+	function should_load() {
+		include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+
+		if ( ! is_plugin_active( 'sassy-social-share/sassy-social-share.php' ) ) {
+			return false;
+		}
+		$ss_options = get_option( 'heateor_sss' );
+		$ss_bars = ['vertical_re_providers', 'horizontal_re_providers'];
+		if ( ! empty( $ss_options ) && is_array( $ss_options ) ) {
```

Comment:
you can return early
```suggestion
		if ( ! is_array( $ss_options ) ) {
		     return false;
		}
```